### ISSUE TITLE
Upgrade deprecated npm packages

### ISSUE THREAD

**Issue by szyn** [treasure-data/digdag#743]
When installing packages with npm, I get the following warnings:

```
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update!
```

It's just a warning, but I think it's a good idea to upgrade the packages! 
If you don't mind, I'll make changes that and create pull requests.

**Comment by hiroyuki-sato** [comment]
@szyn JFYI. #797 has been created. It update `babel-preset-es2015`.

- [x] node-uuid (#744) 
- [ ] babel-preset-es2015: (#797)

**Comment by muga** [comment]
@szyn @hiroyuki-sato sorry, I was not aware of this ticket. We started working on upgrading Node.js v8.x #797. Please take a look and let me know if you have any concerns/suggestions.

**Comment by muga** [comment]
I will merge https://github.com/treasure-data/digdag/pull/797 into master and include it in next version. Please take a look and feel free to comment anywhere when you get a chance.

**Comment by hiroyuki-sato** [comment]
@muga I succeed build dda03afdbd6d68e249d39080729153114f2c3bd7 (merged #797) in my enviromnet. Thank you!

**Comment by muga** [comment]
@hiroyuki-sato thank you for testing. It looks like other warnings are still there though:-)
